src/main.cpp

```diff
@@ -71,6 +72,13 @@ static int inner_main(int argc, char** argv)
     // execution return value
     int retVal = phantom->returnValue();
     delete phantom;
+
+#ifndef NDEBUG
```

Comment:
I don't like them either, but unfortunately, there is no positive QT_DEBUG, only QT_NO_DEBUG in release builds.